sample-functions-framework-java-mvn/pom.xml

```diff
@@ -0,0 +1,45 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google</groupId>
+  <artifactId>sample-functions-java-mvn</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <properties>
+    <maven.compiler.target>11</maven.compiler.target>
```

Comment:
Does this work with Java 1.8?

Reply:
I think 11 is the lowest version supported by the Functions Framework for Java

Reply:
and I don't see Java 8 listed in the docs here: https://cloud.google.com/functions/docs/concepts/exec#runtimes

Reply:
Looks that way: https://github.com/GoogleCloudPlatform/functions-framework-java/blob/master/functions-framework-api/pom.xml#L65-L66